guides/source/ja/engines.md

```diff
@@ -145,6 +145,21 @@ NOTE: `Engine`クラスの定義に含まれる`isolate_namespace`の行を変
 
 `app/controllers`ディレクトリの下には`blorgh`ディレクトリが置かれます。この中には`application_controller.rb`というファイルが1つ置かれます。このファイルはエンジンのコントローラ共通の機能を提供するためのものです。この`blorgh`ディレクトリには、エンジンで使用するその他のコントローラを置きます。これらのファイルを名前空間化されたディレクトリに配置することで、他のエンジンやアプリケーションに同じ名前のコントローラがあっても名前の衝突を避ける事ができます。
 
+NOTE: Rubyが定数を探索する方法のせいで、エンジンのコントローラがエンジンのアプロケーションコントローラを継承するのではなく、メインアプリケーションコントローラを継承する場合があります。これはRubyはすでに`ApplicationController`定数を知っているので自動読み込みが動作されないためです。[定数がトリガーされない場合](autoloading_and_reloading_constants.html#定数がトリガーされない場合)と[定数の自動読み込みと再読み込み](autoloading_and_reloading_constants.html)に詳しい説明があります。この問題を解決する一番良い方法は`require_dependency`を使いエンジンのアプリケーションコントローラがロードされるのを保証することです。例を見ましょう。
+
+``` ruby
+# app/controllers/blorgh/articles_controller.rb:
+require_dependency "blorgh/application_controller"
+
+module Blorgh
+  class ArticlesController < ApplicationController
+    ...
+  end
+end
+```
+
+WARNING: `require`は開発環境でのクラス自動読み込みの誤作動を起こすので使わないでください。`require_dependency`を使用することだけがクラスの読み込みされる、されないを保証する正しい使い方です。
+
```

Comment:
原文:

NOTE: Because of the way that Ruby does constant lookup you may run into a situation
where your engine controller is inheriting from the main application controller and
not your engine's application controller. Ruby is able to resolve the `ApplicationController` constant, and therefore the autoloading mechanism is not triggered. See the section [When Constants Aren't Missed](autoloading_and_reloading_constants.html#when-constants-aren-t-missed) of the [Autoloading and Reloading Constants](autoloading_and_reloading_constants.html) guide for further details. The best way to prevent this from
happening is to use `require_dependency` to ensure that the engine's application
controller is loaded. For example:

``` ruby
# app/controllers/blorgh/articles_controller.rb:
require_dependency "blorgh/application_controller"

module Blorgh
  class ArticlesController < ApplicationController
    ...
  end
end
```

WARNING: Don't use `require` because it will break the automatic reloading of classes
in the development environment - using `require_dependency` ensures that classes are
loaded and unloaded in the correct manner.

Reply:
> `require_dependency`を使用すること **だけ** がクラスの読み込みされる、されないを保証する正しい使い方です。

ニュアンスとしてはあっていそうですが、`only` という単語はないので、「だけ」の部分は外しても良さそうかなと思いました🤔

Reply:
おっとそうでした。　👍